app/controllers/api/v1/accounts/articles_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Api::V1::Accounts::ArticlesController < Api::V1::Accounts::BaseController
+  before_action :portal
+  before_action :fetch_article, except: [:index, :create]
+
+  def index
+    @articles = @portal.articles.where(status: :published)
```

Comment:
I don't think we should limit this to published articles only. This is the admin view, right? It list all the articles in the account.